Tests/Pcap++Test/main.cpp

```diff
@@ -1320,6 +1397,262 @@ PTF_TEST_CASE(TestPcapNgFileReadWriteAdv)
 	writerDev2.close();
 }
 
+PTF_TEST_CASE(TestPcapNgFileReadWriteAdvCompress)

```

Comment:
instead of duplicating the existing test-cases, maybe we can add another writer to the original test-case that will use compression and then check the two writers behave the same

Reply:
Same as above, if you want everything integrated I can do it, it will just make the code a bit messy due to disabled sections woven into the existing test code (if user does not build with zstd)

Reply:
merged